cdap-docs/examples-manual/build.sh

```diff
@@ -140,6 +140,8 @@ function download_includes() {
   test_an_include 31c9d6fd543a48ce5e3f2b9cdc630b6d ../../cdap-examples/FileSetExample/src/main/java/co/cask/cdap/examples/fileset/WordCount.java
   
   test_an_include f2eb96409a39f0cd1cfa09cb7c917946 ../../cdap-examples/HelloWorld/src/main/java/co/cask/cdap/examples/helloworld/HelloWorld.java
+
+  test_an_include a9f268fefd13f019e1c77052ffea1747 ../../cdap-examples/LogAnalysis/src/main/java/co/cask/cdap/examples/loganalysis/LogAnalysisApp.java
```

Comment:
The file hasn't change at all. Weird. I will get the hash and put the right one.